cluster/kubectl.sh

```diff
@@ -32,15 +32,25 @@ set -o pipefail
 
 KUBE_ROOT=${KUBE_ROOT:-$(dirname "${BASH_SOURCE[0]}")/..}
 source "${KUBE_ROOT}/cluster/kube-util.sh"
-source "${KUBE_ROOT}/cluster/clientbin.sh"
```

Comment:
This file is tricky, I'm not sure if we need to use cloud-provider-gcp version, or we are free to use k/k